modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -3,23 +3,39 @@ ifndef::env-site,env-github[]
 
-B2B transactions often communicate contractual obligations between companies. It is therefore desirable (and often necessary) to track transmissions sent or received and store the transmission content as well as metadata. Metadata could be the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing. Anypoint Partner Manager provides a simple, secure, and integrated mechanism for activity tracking within MuleSoft flows. Some uses for this data include:
+B2B transactions often communicate contractual obligations between companies.
+It is desirable (and often necessary) to track transmissions that are sent or received and store the transmission content as well as any metadata.
+Metadata can be the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+Some uses for this data include:
 
 * Non-repudiation.
 * Correlation of acknowledgments.
 * Error correction.
 * Auditing.
 
-Anypoint Partner Manager enables you to monitor B2B activity tracking. Activity is the full set of end-to-end events (including transmissions, documents, and errors) involved in processing a transmission.
+Anypoint Partner Manager allows you to monitor B2B transmission activity.
+Activity is the full set of end-to-end events involved in processing a transmission. This includes transmission metadata, messages, and errors.
 
 You can:
 
-* Filter to find the specific transmission data (for Beta?).
-* Review and monitor errors and acknowledgements.
-* Search for document metadata (for Beta?).
+* Search for document metadata (such as partner name, message type, filename, control number, endpoint, or date received).
+* View and download the payload information from the transmission.
+* Review errors and acknowledgement payloads.
+// * Filter to find the specific transmission flow data.
 
-== To Track Activity:
+== Track Activity for Message Flows:
+
+. Open Partner Manager.
+. From within your Partner Manager environment, select Activity from the left-hand navigation. You see the activity on the message flows you created.
+. Scroll the list. If you don't see the desired transmission then type a partner name, message type, filename, control number, endpoint, or date received in the search box.
+. Select the transmission. You see a summary with the File name, Control Number, Endpoint, and date and time the transmission was received.
+The Transmission Flow section shows the stages of the flow including any acknowledgement you set. If you see an exclamation point in the rectangle box next to one of the stages it means that this is the place where there is an issue with the transmission. Typically, any errors can be viewed in the transmission payload. To inspect the payload:
+
+. Click the *View Payload* button to see the payload transmission.
+. Click the *Download* button to download the payload for analysis.
+. Continue for any other transmission.
```

Comment:
Suggest omitting this. I think it is understood that if you have more than one to check, you repeat the steps.
An alternative is also to keep this, but make it plain text following the last step: Repeat these steps for every transmission that you want to monitor.
If you keep this as a step...:
```suggestion
. Repeat steps to monitor another transmission.
```